benchmarks/routines/quantization.py

```
@@ -0,0 +1,863 @@
+                        )
+                    # Enforce refcheck: fail or warn on mismatches
+                    if num_different_elements > 0:
+                        mismatch_msg = (
+                            f"[mxfp8_quantize] Round-trip mismatch: "
+                            f"{num_different_elements}/{num_elements} "
+                            f"({num_different_elements_percentage:.2f}%) elements differ"
+                        )
+                        if args.allow_output_mismatch:
+                            print(f"[WARNING] {mismatch_msg}")
+                        else:
+                            raise AssertionError(mismatch_msg)
```

Ruff (0.14.11)

Abstract `raise` to an inner function

(TRY301)